_includes/v21.2/sidebar-data/manage.json

```diff
@@ -0,0 +1,517 @@
+        	       "/cockroachcloud/console-access-management.html"
+              ]
+            }
+          ]
+        },
+        {
+          "title": "Securing CockroachDB Self-Hosted Clusters",
+          "items": [
+            {
+              "title": "Overview",
+              "urls": [
+                "/${VERSION}/security-overview.html"
+              ]
+            },
+            {
+              "title": "Authentication",
+              "urls": [
+                "/${VERSION}/authentication.html"
+              ]
+            },
+            {
+              "title": "Encryption",
+              "urls": [
+                "/${VERSION}/encryption.html"
+              ]
+            },
+            {
+              "title": "Authorization",
+              "urls": [
+                "/${VERSION}/authorization.html"
+              ]
+            },
+            {
+              "title": "SQL Audit Logging",
+              "urls": [
+                "/${VERSION}/sql-audit-logging.html"
+              ]
+            },
+            {
+              "title": "GSSAPI Authentication",
+              "urls": [
+                "/${VERSION}/gssapi_authentication.html"
+              ]
+            },
+            {
+              "title": "Single Sign-on",
+              "urls": [
+                "/${VERSION}/sso.html"
+              ]
+            },
+            {
+              "title": "Rotate Security Certificates",
+              "urls": [
+                "/${VERSION}/rotate-certificates.html"
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "title": "Monitoring and Alerting",
+      "items": [
+        {
+          "title": "Monitor Your CockroachDB Cloud Cluster",
+          "items": [
+            {
+              "title": "Cluster Overview Page",
+              "urls": [
+                "/cockroachcloud/cluster-overview-page.html"
+              ]
+            },
+            {
+              "title": "Alerts Page",
+              "urls": [
+                "/cockroachcloud/alerts-page.html"
+              ]
+            },
+            {
+              "title": "Monitoring Page",
+              "urls": [
+                "/cockroachcloud/monitoring-page.html"
+              ]
+            },
+            {
+              "title": "Statements Page",
+              "urls": [
+                "/cockroachcloud/statements-page.html"
+              ]
+            },
+            {
+              "title": "Sessions Page",
+              "urls": [
+                "/cockroachcloud/sessions-page.html"
+              ]
+            },
+            {
+              "title": "Transactions Page",
+              "urls": [
+                "/cockroachcloud/transactions-page.html"
+              ]
+            }
+          ]
+        },
+        {
+          "title": "Monitor Your CockroachDB Self-Hosted Cluster",
+          "items": [
+            {
+              "title": "Overview",
```

Comment:
I don't think we should have multiple titles that simply say Overview.  Can't we qualify them?